frontend/src/features/intake-profile/RenderFields.tsx

```diff
@@ -194,21 +195,24 @@ export const RenderFields = ({
         />
       );
     case 'contact_method':
+
       // eslint-disable-next-line no-case-declarations
-      const {emailFieldId, phoneFieldId} = field.linkedFields;
+      const {emailFieldId, phoneFieldId} = field.linkedFields!;
```

Comment:
Let's revisit this soon to decide precisely the behavior we want, and if the `!` operator meets our needs (as opposed to a more explicit handling of the condition where `linkedFields` is null